sdk/cognitivelanguage/ai-language-conversations/samples-dev/sample_analyze_conversation_app.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to analyze user query for intents and entities using
+ * a conversation project with a language parameter.
+ *
+ * @summary Conversational query analysis for intents and entities extraction
+ * @azsdk-weight 50
+ */
+
+import { ConversationAnalysisClient, ConversationalTask } from "@azure/ai-language-conversations"
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+//Get secrets
+//You will have to set these environment variables for the sample to work
+const clu_endpoint = process.env.AZURE_CONVERSATIONS_ENDPOINT;
+const clu_key = process.env.AZURE_CONVERSATIONS_KEY;
+const project_name = process.env.AZURE_CONVERSATIONS_PROJECT_NAME;
+const deployment_name = process.env.AZURE_CONVERSATIONS_DEPLOYMENT_NAME;
+
+const service: ConversationAnalysisClient = new ConversationAnalysisClient(clu_endpoint, new AzureKeyCredential(clu_key));
+
+const body: ConversationalTask = {
+    "kind": "Conversation",
+    "analysisInput": {
+        "conversationItem": {
+            "id": "id__7863",
+            "participantId": "id__7863",
+            "text": "Send an email to Carol about the tomorrow's demo"
+        }
+    },
+    "parameters": {
+        "projectName": project_name,
+        "deploymentName": deployment_name
+    }
+}
+
+export async function main() {
+    //Analyze query
+    const actionResult = await service.analyzeConversation(body);
+    console.log("query: ", actionResult.result.query);
+    console.log("project kind: ", actionResult.result.prediction.projectKind);
+    console.log("top intent: ", actionResult.result.prediction.topIntent);
+    console.log("category: ", actionResult.result.prediction.intents[0].category);
+    console.log("confidence score: ", actionResult.result.prediction.intents[0].confidence);
+    console.log("entities:");
+
+    actionResult.result.prediction.entities.forEach(entity => {
```

Comment:
Could we skip one access level by destructing the output object as follows?
```suggestion
    const { result } = await service.analyzeConversation(body);
    console.log("query: ", result.query);
    console.log("project kind: ", result.prediction.projectKind);
    console.log("top intent: ", result.prediction.topIntent);
    console.log("category: ", result.prediction.intents[0].category);
    console.log("confidence score: ", result.prediction.intents[0].confidence);
    console.log("entities:");

    result.prediction.entities.forEach(entity => {
```